packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -144,11 +154,45 @@ This can be used only together with build type ${input('release')} for Android a
         },
         {
           name: 'build-file-name',
-          summary: 'The name for the downloaded build file',
+          summary: '(Deprecated) The name for the downloaded build file',
```

Comment:
I think the correct way is to use `MetadataGroup.DEPRECATED` in the groups and not the the description

Reply:
oh yeah